vaadin-notification-flow-parent/vaadin-notification-flow-integration-tests/src/test/java/com/vaadin/flow/component/notification/tests/NotificationWithClassNamesIT.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.notification.tests;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import com.vaadin.flow.component.notification.testbench.NotificationElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.tests.AbstractComponentIT;
+
+@TestPath("vaadin-notification/notification-class-names-test")
+public class NotificationWithClassNamesIT extends AbstractComponentIT {
+
+    public static final String NOTIFICATION_CARD = "vaadin-notification-card";
+
+    @Before
+    public void init() {
+        open();
+    }
+
+    @Test
+    public void openNotification_cardHasSameClassNames() {
+        findElement(By.id("open-notification-btn")).click();
```

Comment:
Consider using the initialization block, e.g.

```java
    @Before
    public void init() {
        open();
        openNotification = $(ButtonElement.class).id("open-notification-btn");
        // other buttons...
    }
```

This should make the tests more readable.